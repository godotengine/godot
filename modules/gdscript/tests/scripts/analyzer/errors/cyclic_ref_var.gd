func test():
	print(v1)

var v1 := v2
var v2 := v1
