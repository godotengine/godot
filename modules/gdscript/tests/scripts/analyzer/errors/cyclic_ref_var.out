GDTEST_ANALYZER_ERROR
>> ERROR at line 5: Could not resolve member "v1": Cyclic reference.
>> ERROR at line 5: Cannot infer the type of "v2" variable because the value doesn't have a set type.
